numpy/distutils/misc_util.py

```diff
@@ -2295,6 +2295,41 @@ def get_info(name):
     return g.get(name, g.get(name + "_info", {}))
 
 def show():
+    """Print information about various resources (libraries, library directories,
+    include directories, etc.) in the system on which NumPy was built.
+
+    Information is grouped according to classes (``blas_opt_info`` in the example)
+    that specify relevant resources, and it is formatted to be compatible with
+    ``distutils.setup`` keyword arguments.
```

Comment:
I don't think this last sentence is correct, there are no such keywords for ``setup``. They are there for ``add_library`` and ``add_extension`` though, so I guess that's what you meant. It's not super important I think, so suggest moving to the end of the Notes section.